server/api/rounds.py

```diff
@@ -670,7 +671,13 @@ def create_round(election: Election):
             AuditType.BALLOT_COMPARISON: "supersimple",
         }[AuditType(election.audit_type)]
         sample_sizes = {
-            contest_id: options[sample_size_key]["size"]
+            contest_id: (
+                (sample_size_options[contest_id]).get(
+                    sample_size_key, sample_size_options[contest_id]["asn"]
+                )["size"]
+                if election.audit_type == AuditType.BALLOT_POLLING
+                else sample_size_options[contest_id][sample_size_key]["size"]
+            )
```

Comment:
I think this might be a little cleaner to read like this:

```python
sample_size_key = {
  AuditType.BALLOT_POLLING: "0.9" if "0.9" in sample_size_options[contest_id] else "asn",
  ...
}
```
and then the rest can stay the same as it was before

Reply:
See I knew there was a fancy syntax way to do it, I just couldn't figure it out.